lib/mayaUsd/CMakeLists.txt

```diff
@@ -176,4 +176,5 @@ add_subdirectory(utils)
 
 if(BUILD_HDMAYA)
     add_subdirectory(render/mayaToHydra)
+    add_dependencies(mtoh hdMaya)
```

Comment:
Why? How is that connected to the new API you are adding?

Reply:
It isn't. It's connected to the api you've added, and a fix I've had to add to fix it?

Reply:
It isn't. It's connected to the api you've added, and a fix I've had to add to fix it?

Reply:
@robthebloke  `add_dependencies(mtoh hdMaya)` is not really needed. Once you sync your branch with latest dev, you should get this change automatically:

https://github.com/Autodesk/maya-usd/commit/5eab9f4cc57f9563e07f176624bd8dba421e709c